Makefile

```
@@ -39,6 +46,16 @@ test:
 	@docker build --target=test -t datamegh:test .
 	@docker run datamegh:test
 
+.PHONY: all test clean
+
+tag:
+	@echo "Bump version :- $(last_tag) -> $(new_version)"
+	@sed -i "s/version.*=.*/version = '$(new_version)'/" datamegh/__init__.py && \
+		git add datamegh/__init__.py && \
+		git commit -m "Update $(last_tag) to $(new_version)" -m "[skip ci]" && \
```

No need previous version in the commit message.
```suggestion
		git commit -m "Update version to $(new_version)" -m "[skip ci]" && \
```